integration/helpers/cf-template/package.json

```diff
@@ -1,29 +1,30 @@
 {
-  "name": "remix-template-cloudflare-workers",
+  "name": "integration-cf-template",
+  "version": "0.0.0",
   "private": true,
   "sideEffects": false,
   "type": "module",
   "scripts": {
-    "build": "node ../../../build/node_modules/@remix-run/dev/dist/cli.js build",
-    "dev": "node ../../../build/node_modules/@remix-run/dev/dist/cli.js dev --manual -c \"npm start\"",
-    "start": "node ../../../node_modules/wrangler/bin/wrangler.js dev ./build/index.js"
+    "build": "node ./node_modules/@remix-run/dev/dist/cli.js build",
```

Comment:
not 100% sure on this one, but I believe if you use `pnpm remix build` this will do the same thing e.g. point to linked cli package

Reply:
if that doesn't work, the next thing I'd try is using `pnpm exec`

Reply:
Oh, nice one.

Reply:
I tried this and unfortunately this doesn't work with our current setup because the package bins don't exist at the time of `pnpm install`.